includes/Modules/Analytics.php

```diff
@@ -411,6 +421,28 @@ protected function enqueue_gtag_js( $property_id ) {
 				'gtag(\'config\', \'' . esc_attr( $property_id ) . '\', ' . wp_json_encode( $gtag_opt ) . ' );'
 			);
 		}
+
+		$block_on_consent_attrs = $this->get_tag_block_on_consent_attribute();
+
+		if ( $block_on_consent_attrs ) {
+			$apply_block_on_consent_attrs = function ( $tag, $handle ) use ( $block_on_consent_attrs, $gtag_src ) {
+				if ( 'google_gtagjs' !== $handle ) {
+					return $tag;
+				}
+
+				return str_replace(
+					array(
+						"<script src='$gtag_src", // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+						"<script src=\"$gtag_src\"", // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+						"<script type='text/javascript' src='$gtag_src", // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
+						"<script type=\"text/javascript\" src=\"$gtag_src", // phpcs:ignore WordPress.WP.EnqueuedResources.NonEnqueuedScript
```

Comment:
The presence of the `type` attribute depends on the current theme's declared support for `html5` so we have to account for both cases here. Both quote types are supported as well just to be safe.

Also, the reason this matches the `src` at all is because secondary extra/inline scripts associated with the handle are also included in `$tag` so we only want to change the main script.